projects/recipe-page/phase-4-DOM-Manipulation.md

```diff
@@ -44,7 +44,9 @@ document.querySelector('ul').appendChild(newListItem);
 
 #### 2. Adding a check mark as an element to a list
 
-To add checkbox to the page, we’ll need to use a this process
+To add checkbox to the page, we’ll need to use this process:
+
+(n.b. 'var' is now deprecated, so use either 'let' or 'const' to create a variable)
```

Comment:
I believe `var` is not technically deprecated in JavaScript, but it's strongly recommended to use let and const instead in modern JavaScript development. Should we describe it in this way?

Reply:
I will adjust the note to explain that it is not deprecated for backward compatibility reasoning but it is now advised to use let or consst. Thank you @thisIsTheWei !!

Reply:
https://github.com/Techtonica/curriculum/pull/2193/commits/21662f9c5ebcb94c7ce551a068c5ef50fa869779 @thisIsTheWei

Reply:
Great work!